schema/elasticsearch/visibility/versioned/v7/index_template_v7.json

```diff
@@ -92,6 +92,9 @@
       },
       "RootRunId": {
         "type": "keyword"
+      },
+      "PausedEntities": {
+        "type": "keyword"
```

Comment:
Ugh. I'm sorry, you did the right thing before by creating the `v8` folder. I totally got confused.

Reply:
done